debian/control

```diff
@@ -0,0 +1,15 @@
+Source: ondemand
+Section: unknown
+Priority: optional
+Maintainer: root <>
+Build-Depends: debhelper (>=11~), ruby
```

Comment:
I would imagine also need at least `nodejs` here too, along with a long list of other things to build mostly native gems like `make` and `gcc`, not sure if can just depend on build essentials here.